framework/resources/Datasources/table_stats/analyze_tables.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+source conf/drillTestConfig.properties
+
+set -xe
+
+
+${DRILL_HOME}/bin/sqlline -n ${USERNAME} -p ${PASSWORD} -u "jdbc:drill:schema=dfs.drillTestDir;drillbit=${DRILL_STORAGE_PLUGIN_SERVER}"  --run=${DRILL_TEST_DATA_DIR}/Datasources/table_stats/analyze_tables.sql
```

Comment:
Can you add a check to see if password is defined ( as its optional). 

Like this..
if [ -z "$PASSWORD" ]
then
  <statement>
else
  <statement without password>
fi

See https://github.com/mapr/drill-test-framework/blob/master/framework/resources/Datasources/tpcds/createTablesJson.sh